skype/skype-ps/skype/New-CsOnlineApplicationInstance.md

```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: New-CsOnlineApplicationInstance
+schema: 2.0.0
+author: kenwith
+ms.author: kenwith
+ms.reviewer:
+---
+
+# New-CsOnlineApplicationInstance
+
+## SYNOPSIS
+Creates an application instance in AAD. The application instance will be sync'ed into LyncAD and BVD after it's created.
+
+## SYNTAX
+
+```
+New-CsOnlineApplicationInstance [-UserPrincipalName] <string> [[-ApplicationId] <guid>] [[-DisplayName] <string>]
+    [-Tenant <guid>] [-Force] [-WhatIf] [-Confirm]  [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet is used to create an application instance in AAD. The application instance will be sync'ed into LyncAD and BVD after it's created.
+
+## EXAMPLES
+
+### -------------------------- Example 1 --------------------------
+```powershell
+New-CsOnlineApplicationInstance -UserPrincipalName appinstance01@contoso.com -ApplicationId 00000000-0000-0000-0000-000000000000 -DisplayName "AppInstance01" -Tenant 00000000-0000-0000-0000-000000000001
```

Comment:
Please do not use all 0's guid in example (maybe just generate one).
No need to provide tenant parameter